lib/onfido/api.rb

```diff
@@ -2,48 +2,52 @@
 
 module Onfido
   class API
-    def initialize(options = {})
-      @api_key = options[:api_key]
+    def initialize(api_key:, region:, **extra_options)
+      @options = Onfido::Options.new(api_key: api_key, region: region, **extra_options)
```

Comment:
This `Onfido::Options` class is just used internally to store all the options and to make testing a bit easier.